examples/3d-heatmap/webpack.config.js

```diff
@@ -38,6 +38,5 @@ module.exports = {
   ]
 };
 
-// DELETE THIS LINE WHEN COPYING THIS EXAMPLE FOLDER OUTSIDE OF DECK.GL
-// It enables bundling against src in this repo rather than installed deck.gl module
-module.exports = require('../webpack.config.local')(module.exports);
+// This line enables bundling against src in this repo rather than installed deck.gl module
+module.exports = env => env ? require('../webpack.config.local')(CONFIG)(env) : CONFIG;
```

Comment:
`npm run build` won't work if you copy this example out of the deck.gl directory. Is this intentional?